### PR TITLE
Dynamic simulation documentation

### PR DESCRIPTION
**Pull request by marcosmc**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*
#108 

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
docs update: Add dynamic-simulation page

### CODE REVIEW

File: docs/tools/dynamic-simulation.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+---
+
+The `dynamic-simulation` command is used to run dynamic simulations on the network.
+
+# Usage
+```shell
+$> itools dynamic-simulation --help
+usage: itools [OPTIONS] dynamic-simulation --case-file <FILE> [--help] [-I <property=value>]
+              [--import-parameters <IMPORT_PARAMETERS>] [--output-file <FILE>]
+              [--parameters-file <FILE>] [--skip-postproc]
```
Comment (agnesLeroy) [inline docs/tools/dynamic-simulation.md:13]: the list of parameters and their description should be updated based on the specifications document.

File: docs/tools/dynamic-simulation.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+# Configuration
+To run a dynamic simulation, one has to choose the implementation, follow the instructions at [dynamic simulation Configuration](../configuration/modules/dynamic-simulation.md).
+
+To set the default configuration of the dynamic simulation, one has to configure the
+[dynamic-simulation-default-parameters](../configuration/modules/dynamic-simulation-default-parameters.md) module.
+
+To learn more about configuration files, read the [DynamicSimulationParameters](../configuration/parameters/DynamicSimulationParameters.md) documentation
+page.
+
+# Examples
```
Comment (agnesLeroy) [inline docs/tools/dynamic-simulation.md:64]: The examples should also be modified based on the specifications.

File: pages/documentation/simulation/timedomain/dynawo.md [hunk 3]
```diff
@@ -5,3 +5,98 @@ layout: default
 # Dynawo
 
 Powsybl provides an implementation to run dynamic simulations with [Dynawo](https://dynawo.github.io/).
+
+## Configuration
```
Comment (mathbagu) [inline pages/documentation/simulation/timedomain/dynawo.md:9]: Add a section above to explain how to get and install dynawo. Maybe we could git a link to the website or the readme where it's already explained

Comment (gautierbureau) [reply pages/documentation/simulation/timedomain/dynawo.md:9]: One would need to install a distribution, https://github.com/dynawo/dynawo#distribution or https://dynawo.github.io/install/

Comment (marcosmc) [reply pages/documentation/simulation/timedomain/dynawo.md:13]: We have added a new section with a reference and a link to the website with the Dynawo distributions [https://dynawo.github.io/install/](https://dynawo.github.io/install/)

**Review by mathbagu — APPROVED** [review]

**Pull request merged**